src/pages/strom/reset-sent/index.tsx

```diff
@@ -0,0 +1,15 @@
+import {NextPage} from 'next'
+
+import {PasswordResetSent} from '@/components/PasswordResetSent/PasswordResetSent'
+
+import {PageLayout} from '../../../components/PageLayout/PageLayout'
```

Comment:
```suggestion
import {PageLayout} from '@/components/PageLayout/PageLayout'
```

akoze asi to si skopiroval z niekade ale toto by bolo vhodnejsie